### PR TITLE
467 init command phase 2

### PR DESCRIPTION
**Pull request by tnolet**
I hereby confirm that I followed the code guidelines found at [engineering guidelines](https://www.notion.so/checkly/Engineering-Guidelines-a3a165a203a04dc1a112f0e26b2f2d3f)

### Notes for the reviewer

- adds hints, header and footers to the terminal UX.
- adds the option to specify a template using `npm create checkly -- --template <template>`
- makes examples TS only
- adds boilerplate template
- removes TS selection step in wizard

### How to test?

- pull branch
- run `npm start` in `packages/create-cli`

a typical session would look like:

CleanShot 2023-02-03 at 15 09 43

### New dependency submission
- boilerplate-project

### CODE REVIEW

File: examples/simple-project/src/services/docs/__checks__/docs-search.spec.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
+"use strict";
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {
```
Comment (clample) [inline examples/simple-project/src/services/docs/__checks__/docs-search.spec.js:2]: I think that the compiled TS files were accidentally included in the PR

Comment (tnolet) [reply examples/simple-project/src/services/docs/__checks__/docs-search.spec.js]: fixed

File: packages/create-cli/src/utils/__tests__/directory.spec.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"use strict";
```
Comment (clample) [inline packages/create-cli/src/utils/__tests__/directory.spec.js:1]: I think that this is some compiled TS that could be deleted as well

Comment (tnolet) [reply packages/create-cli/src/utils/__tests__/directory.spec.js]: yep, seems like in the last 3 hours my IDE started storing these buggers.

**Review by clample — APPROVED** [review]

File: packages/create-cli/src/commands/bootstrap.ts [hunk 3]
```diff
@@ -43,44 +75,46 @@ export default class Bootstrap extends Command {
...
+    const templateResponse = await prompts({
       type: 'select',
       name: 'template',
-      message: 'How would you like to setup your new project?',
+      message: 'Which template would you like to use for your new project',
       choices: [
-        { value: 'simple-project', title: 'A simple project with a set of best practices (recommended)' },
-        { value: 'empty-project', title: 'An empty project with basic config' },
+        { value: 'simple-project', title: 'A simple project template with a set of best practices (recommended)' },
+        { value: 'boilerplate-project', title: 'A boilerplate project with basic config' },
```
[the remark this answers is not in the document]
Comment (tnolet) [reply packages/create-cli/src/commands/bootstrap.ts:96]: yes, that makes total sense. The simple project is also not that simple anymore

[the remark this answers is not in the document]
Comment (clample) [reply packages/create-cli/src/commands/bootstrap.ts:96]: > The simple project is also not that simple anymore

It's grown quite a bit 😄

**Review by clample — APPROVED** [review]

**Pull request merged**